frontend/src/Explorer/View/Layout.purs

```diff
@@ -24,17 +25,18 @@ view state =
     let mobileMenuClazz = if state ^. (viewStates <<< globalViewState <<< gViewMobileMenuOpenend)
                           then " mobile__menu--opened"
                           else ""
+        routeClazz = CSS.route $ state ^. route
     in
     P.div
-      [ P.className $ "explorer-container" <> mobileMenuClazz ]
+      [ P.className $ "explorer-container" <> mobileMenuClazz <> " " <> routeClazz ]
```

Comment:
This looks very fragile. I would much rather wanted to see:
```
P.className $ String.joinWith " " ["explorer-container", mobileMenuClazz, routeClazz]
```

It makes clear this way we won't accidentally concatenate two classes.